frontend/src/components/shared/ComparisonMatrix.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div v-if="mergedComparisons" class="table-container">
-    <table class="table">
+    <table class="table comparison-matrix">
```

Comment:
I don't have the stack running at the moment but I think the `table` class still serves a purpose here since it has some Bulma styling. See: https://bulma.io/documentation/elements/table/

Reply:
I see. Although I don't see any visual differences with and without, to be safe, we should keep it.